### PR TITLE
Merge change from 1.0.0 to 2.0.0

### PR DESCRIPTION
**Pull request by vshand11**
Merge change from 1.0.0 to 2.0.0, solve conflicts and conduct tests.

### CODE REVIEW

File: src/essential_validator.py [hunk 1]
```diff
@@ -431,6 +479,7 @@ def validate_data(self, file_info):
     validate many to many relationship
     """
     def check_m2m_relationship(self, columns, duplicate_ids, id_field, rel_props, file_info):
+        rtn_val = True
```
Comment (n2iw) [inline src/essential_validator.py:482]: Looks same as line 483

File: src/metadata_export.py [hunk 2]
```diff
@@ -30,6 +30,7 @@ def metadata_export(configs, job_queue, mongo_dao):
         # dump models to json files
         dump_dict_to_json(model_store.models, f"models/data_model.json")
     except Exception as e:
+        log.exception(e)
```
Comment (n2iw) [inline src/metadata_export.py:33]: Same as line 34?

File: src/metadata_export.py [hunk 3]
```diff
@@ -200,6 +201,7 @@ def export(self, submission_id, node_type):
                     self.log.info(f"{submission_id}: {count + start_index} {node_type} nodes are exported.")
                     return
                 except Exception as e:
+                    self.log.exception(e)
```
Comment (n2iw) [inline src/metadata_export.py:204]: Same as line 205?

File: src/essential_validator.py [hunk 4]
```diff
@@ -35,6 +35,7 @@ def essentialValidate(configs, job_queue, mongo_dao):
         # dump_dict_to_json([model[MODEL] for model in model_store.models], f"tmp/data_models_dump.json")
         dump_dict_to_json(model_store.models, f"models/data_model.json")
     except Exception as e:
+        log.exception(e)
```
[the remark this answers is not in the document]
Comment (vshand11) [reply src/essential_validator.py]: updated

File: src/essential_validator.py (hunk 1 above)
Comment (vshand11) [reply src/essential_validator.py]: Updated

File: src/metadata_export.py (hunk 2 above)
Comment (vshand11) [reply src/metadata_export.py]: Updated

File: src/metadata_export.py (hunk 3 above)
Comment (vshand11) [reply src/metadata_export.py]: updated.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**